packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -195,6 +196,7 @@ const defaultOptions = {
   showNativePhotoCameraButton: false,
   showNativeVideoCameraButton: false,
   theme: 'light',
+  autoOpen: false,
```

Comment:
Since the option is optional we can use `undefined` to act as `false` and make the type simpler.

```suggestion
```

Reply:
I think we should keep `false` as an available option, allows the user to do the explicit `autoOpen=false` instead of doing `autoOpen=undefined`/deleting the value from the object.

Reply:
I think I would still prefer simpler types/options. @mifi confirmed this too in Slack.